doc/fuzzing.md

```diff
@@ -153,13 +153,17 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
-Full configuration step that was tested on macOS with `brew` installed `llvm`:
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CMAKE_EXE_LINKER_FLAG `-fuse-ld=lld`.
+
+Full configuration step for macOS:
```

Comment:
It's not exactly *full*, we still need:
```bash
$ cmake --build build_fuzz -j$(nproc)
$ FUZZ=process_message build_fuzz/src/test/fuzz/fuzz
```

Reply:
Technically, it is the full _configuration_ step :)